core/common/buffer_or_view.hpp

```diff
@@ -28,8 +28,7 @@ namespace kagome::common {
     BufferOrView(const BufferView &view) : variant{view} {}
 
     template <size_t N>
-    BufferOrView(const std::array<uint8_t, N> &array)
-        : variant{gsl::make_span(array)} {}
+    BufferOrView(const std::array<uint8_t, N> &array) : variant{array} {}
```

Comment:
Ambigous constructor
```suggestion
    BufferOrView(const std::array<uint8_t, N> &array) : variant{BufferView{array}} {}
```